tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -99,35 +99,47 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
 
 ## Configure GCS access
 
-To allow TiDB cloud to access the source data in your GCS bucket, you need to configure the GCS access for each TiDB Cloud as a service on the GCP project and GCS bucket pair. Once the configuration is done for one cluster in a project, all database clusters in that project can access the GCS bucket.
+To allow TiDB Cloud to access the source data in your GCS bucket, you need to configure the GCS access for each TiDB Cloud as a service on the GCP project and GCS bucket pair. Once the configuration is done for one cluster in a project, all database clusters in that project can access the GCS bucket.
 
-1. Get the Google Cloud Service Account ID of the target TiDB cluster.
+1. In the TiDB Cloud console, get the Google Cloud Service Account ID of the target TiDB cluster.
 
-    1. In the TiDB Cloud console, locate your target cluster on the **Clusters** page, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
+    1. In the TiDB Cloud console, choose your target project, and navigate to the **Clusters** page.
+
+    2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
+
+        > **Tip:**
+        >
+        > Alternatively, you can also click the name of your cluster on the **Clusters** page and click **Import Data** in the **Import** area.
+
+    3. On the **Data Import** page, click **Show Google Cloud Service Account ID**, and then copy the Service Account ID for later use.
+
+2. In the Google Cloud Platform (GCP) Management Console, create an IAM role for your GCS bucket.
 
-    2. Click **Show Google Cloud Service Account ID**, and then copy the Service Account ID.
+    1. Sign in to the [GCP Management Console](https://console.cloud.google.com/).
+    2. Go to the [Roles](https://console.cloud.google.com/iam-admin/roles) page, and then click **CREATE ROLE**.
 
-2. In the Google Cloud Platform (GCP) Management Console, go to **IAM & Admin** > **Roles**, and then check whether a role with the following read-only permissions of the storage container exists.
+        ![Create a role](/media/tidb-cloud/gcp-create-role.png)
 
-    - storage.buckets.get
-    - storage.objects.get
-    - storage.objects.list
+    3. Enter a name, description, ID, and role launch stage for the role. The role name cannot be changed after the role is created.
+    4. Click **ADD PERMISSIONS**.
+    5. Add the following read-only permissions to the role, and then click **Add**.
 
-    If yes, you can use the matched role for the target TiDB cluster in the following steps. If not, go to **IAM & Admin** > **Roles** > **CREATE ROLE** to define a role for the target TiDB cluster.
+        - storage.buckets.get
+        - storage.objects.get
+        - storage.objects.list
```

Comment:
- storage.buckets.get
        - storage.objects.get
        - storage.objects.list
        can't add  the three permission directly , it  needs to  select 2 different role then check the corresponding permission respectively. Refer to the document and  search the custom role page https://pingcap.feishu.cn/docx/doxcnQcU3KPOHvIMpTgcgB7kKKb

Reply:
Added " You can copy a permission name to the **Enter property name or value** field as a filter query, and choose the name in the filter result. To add the three permissions, you can use **OR** between the permission names."